app/src/main/java/de/westnordost/streetcomplete/data/download/MobileDataAutoDownloadStrategy.java

```diff
@@ -37,4 +37,9 @@
 	{
 		return 600;
 	}
+
+	@Override
+	protected int getMinQuestsTypesInActiveRadius() {
+		return 2;
+	}
```

Comment:
I think 2 will not make much of a difference to the current behavior. The auto downloader currently downloads 4 quest types until it checks again if there are now enough quests.

Question is, why is the current behavior not already enough to ensure a minimum variation in quest types? To find the answer to this question is important to know what should be changed to better the behavior.

Reply:
> Question is, why is the current behavior not already enough to ensure a minimum variation in quest types? To find the answer to this question is important to know what should be changed in order to better the behavior.

I encountered case where multiple downloaded quests generated 0 results and road name quest generated big cluster of markers.

It was bad in two ways - (1) cluster was close geographically but not in place where someone wanted to walk (2) tester believed that app will continue to ask solely about street names.

Reply:
ok